enclave_manager/avalon_enclave_manager/attestation/epid_enclave_info.i

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2020 Intel Corporation
```

Comment:
Is this SWIG interface file specific to EPID ? The header file included in this files doesn't suggest that this is specific to epid.  If this file will be common for EPID/DCAP we can name it `enclave_info.i`. If it's specific to EPID then we need to relook/rename into the header files included in this file (eg: Is `enclave_info.h` EPID specific. If so we can rename it to `epid_enclave_info.h`). 

I'm expecting some contents in file to be shared by both EPID and DCAP. In that case we can have `enclave_info.i` , `epid_enclave_info.i` and `dcap_enclave_info.i`

Reply:
I will create different headers in DCAP changes PR.